Formula/scm-manager.rb

```diff
@@ -13,12 +13,12 @@ class ScmManager < Formula
     sha256 "10f94fa5dafdb40dbcf6a7744a98b2433e294af16e7712a572de49daaed031e0" => :mavericks
   end
 
-  depends_on :java => "1.6+"
+  depends_on :java => "1.7+"
```

Comment:
Note that Java 7 is now required per https://bitbucket.org/sdorra/scm-manager/wiki/release-notes